src/bot/commands/lastfm/fm.ts

```diff
@@ -164,4 +259,45 @@ export class LastFm extends Command {
 
         await db.save();
```

Comment:
Maybe I missed it, but did you modify the database anywhere that it needs to be saved?